front/app/api/idea_statuses/keys.ts

```diff
@@ -0,0 +1,13 @@
+const ideaStatusesKeys = {
```

Comment:
Keys are defined for each endpoint. The `type` should correspond to the backend type for the caching mechanisms to work correctly.